VideoQuickStart/ViewController.swift

```diff
@@ -404,6 +482,34 @@ extension ViewController : TVIRemoteParticipantDelegate {
         logMessage(messageText: "Unsubscribed from audio track for Participant \(participant.identity)")
     }
     
+    func subscribed(to dataTrack: TVIRemoteDataTrack,
+                    publication: TVIRemoteDataTrackPublication,
+                    for participant: TVIRemoteParticipant) {
+
+        // We are subscribed to the remote Participant's data Track. We will start receiving the
+        // remote Participant's data messages now.
+
+        logMessage(messageText: "Subscribed to data track for Participant \(participant.identity)")
+        
+        if let remoteDataTrack = publication.remoteTrack {
+            remoteDataTrack.delegate = self
+        }
```

Comment:
Use the nonnull `dataTrack` argument instead? Same question for unsubscribed.

Reply:
Derp. Good catch.